cli/core/mpt/flows.py

```
@@ -55,11 +60,12 @@ def search_uom_by_name(mpt_client: MPTClient, uom_name: str) -> Uom:
     """
-    response = mpt_client.get(f"/catalog/units-of-measure?name={uom_name}&limit=1&offset=0")
-    response.raise_for_status()
+    name_query = RQLQuery(name=uom_name)
+    uom_collection = mpt_client.catalog.units_of_measure
+    uom_data_response = uom_collection.filter(name_query).select()
+    uom_result = list(uom_data_response.iterate())
 
-    response_data = response.json()["data"]
-    if not response_data:
+    if not uom_result:
         raise MPTAPIError(f"Unit of measure by name '{uom_name}' is not found.", "404 not found")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)